tests/tcIoc_motion.py

```diff
@@ -140,12 +144,23 @@ def test_WHEN_limits_hit_THEN_motor_reports_limits(self, _, motor_pv_suffix, pv_
     def test_WHEN_axis_set_up_THEN_retries_are_not_allowed(self):
         self.ca.assert_that_pv_is(MOTOR_RTRY, 0)
 
-    def test_WHEN_enabled_set_on_plc_THEN_motor_able_is_set_correctly(self):
+    def test_WHEN_enabled_set_on_plc_THEN_motor_status_on_is_set_correctly(self):
         axis_num = 1
         self.ca.set_pv_value(ENABLE.format(axis_num), 0)
-        self.ca.assert_that_pv_is(MOTOR_ABLE_BASE.format(axis_num), 'Disable') # values are flipped vs PLC, so _able == 1 is PLC disabled etc
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 0)
+
         self.ca.set_pv_value(ENABLE.format(axis_num), 1)
-        self.ca.assert_that_pv_is(MOTOR_ABLE_BASE.format(axis_num), 'Enable')
+        self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), 1)
```

Comment:
If I'm understanding this test correctly, I think this be broken into two tests using Parameterised to test enabled and disabled separately. @rerpha do you agree?

Reply:
Could do, but i'm worried that if it's enabled by default and the test just sets 1 to MOTOR_ON it'll pass and give a false positive because the default value will match the expected one

Reply:
That' a valid point. I would be happy to either leave this as is, or maybe we could set initial state which is opposite to the state we are intending to test the setting of as part of the test - something like the below example which would still achieve breaking the tests into two separate tests so we are testing as atomic items as best a we can:

```python

@parameterized.expand(parameterized_list([
    ("Mode_On", 1, 0),
    ("Mode_Off", 0, 1),
]))
def test_WHEN_enabled_set_on_plc_THEN_motor_status_on_is_set_correctly(self, _, initial_state, desired_state):
    axis_num = 1

    self.ca.set_pv_value(ENABLE.format(axis_num), initial_state) # Set Initial State - Opposite to state we are testing
    self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), initial_state)  # Check Initial State correctly set

    self.ca.set_pv_value(ENABLE.format(axis_num), desired_state)  # Set Desired State
    self.ca.assert_that_pv_is(MOTOR_ON_STATUS.format(axis_num), desired_state)  # Check Desired State correctly set
```

Reply:
looks good, will do that for both